### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by Whoistolu**
# Awsome Books

## In this milestone, we:
- Created a collection that keeps a list of books.
- Created a function to add a new book to the collection, with title and author.
- Created a function to remove a book from the collection.
- Displayed all books saved in the collection in the top part of the page.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -32,20 +32,6 @@ jobs:
...
-          node-version: "12.x"
-      - name: Setup Stylelint
-        run: |
-          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
-          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.stylelintrc.json
-      - name: Stylelint Report
-        run: npx stylelint "**/*.{css,scss}"
   eslint:
     name: ESLint
     runs-on: ubuntu-18.04
```
Comment (carlos-ssh) [inline .github/workflows/linters.yml]: - Kindly enable your stylelint from your [linters.](https://github.com/microverseinc/linters-config/tree/master/html-css-js).
- Fix your MIT License link.
- Your package.json file it's missing the scripts for build and run the server.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    let html = '';
+    convertedBooks.forEach((book) => {
+      html += `<article>
+     <h2>${book.title}</h2>
+     <h2>${book.author}</h2>
+     <button data-book-id = "${book.id}" id = "remove-button" onclick="removeBook(${book.id})">Remove</button>
+    
+   </article></br>`;
+    });
+    // document.getElementById('bookstatus').innerHTML = '';
```
Comment (carlos-ssh) [inline index.js]: - Remove unused lines of code.
- Kindly add a function that removes a book from the collection.
- Provide a function that displays all books saved in the collection in the top part of the page.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi Team, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 1 above)
Comment (Whoistolu) [reply .github/workflows/linters.yml]: Hello @carlos-ssh,
Thank you for your feedback. 
We have fixed the issues as recommended.

File: index.js (hunk 2 above)
Comment (Whoistolu) [reply index.js]: Hello @carlos-ssh,
Thank you for your feedback. 
We have fixed the issues as recommended.

File: index.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    const convertedBooks = JSON.stringify(tempBooks);
+    localStorage.setItem('booksData', convertedBooks);
+    window.location.reload();
+  }
+};
+
+document.getElementById('addbook').addEventListener('click', addBook);
+
+// eslint-disable-next-line no-unused-vars
+function removeBook(id) {
```
Comment (rloterh) [inline index.js:24]: - [x] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

File: .github/workflows/linters.yml (hunk 1 above)
Comment (rloterh) [reply .github/workflows/linters.yml]: This issue is well resolved! ✔️

File: index.js (hunk 2 above)
Comment (rloterh) [reply index.js]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Toluwase & Niko, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾:shipit:.
Congratulations! 🎉🥂

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comment(s) marked [OPTIONAL] under this review:_

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 3 above)
Comment (Whoistolu) [reply index.js:24]: Thank you for your feedback @rlorerh.
This is noted.

**Pull request merged**